Lib/asyncio/events.py

```diff
@@ -672,6 +672,26 @@ def get_event_loop(self):
 
         Returns an instance of EventLoop or raises an exception.
         """
+        if (self._local._loop is None and
+                not self._local._set_called and
+                threading.current_thread() is threading.main_thread()):
+            stacklevel = 2
+            try:
+                f = sys._getframe(1)
+            except AttributeError:
+                pass
+            else:
+                while f:
+                    module = f.f_globals.get('__name__')
+                    if not (module == 'asyncio' or module.startswith('asyncio.')):
+                        break
+                    f = f.f_back
+                    stacklevel += 1
+            import warnings
```

Comment:
I see no reason to delay this import, since it's a toplevel import in base_events.py and proactor_events.py -- by the time this code runs those have been imported anyway.

Reply:
I prefer to only import `warnings` when it is used, so the code which does not use any deprecated feature will not need to import it. It also save us from forgetting to remove the import when warnings are replaced with errors. I think that we should do this in `base_events.py` and `proactor_events.py`.

Reply:
Okay.